File: api/code-samples/python/klay/block/getHeaderByNumber.py
import json
from opensdk.sdk import OpenSDK

host = "https://api.baobab.klaytn.net:8651"

blockTag = "0x1b4"

sdk = OpenSDK(host)
klay_response = sdk.klay.get_header_by_number(blockTag)

print(json.loads(klay_response.response.data))

File: web3rpc/rpc-specs/code-samples/javascript/klay/block/getHeaderByNumber.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
let sdk = new JsonRpcProvider("https://public-en-baobab.klaytn.net");

    const blockNumber = 1
    
    sdk.klay.getHeaderByNumber(blockNumber, {}, (err, data, response) => {
        console.log(data);
    });

}
)()